fontforge/python.c

```diff
@@ -2391,33 +2425,31 @@ static Py_ssize_t PyFFContour_Length( PyObject *self ) {
 return( ((PyFF_Contour *) self)->pt_cnt );
 }
 
+static PyFF_Contour *PyFFPointList_Parse(PyObject *args);
```

Comment:
These functions call each other, but it's safe because the _Concat functions never call PointList_Parse when called with a second contour.